### ISSUE TITLE
support for Node v20

### ISSUE THREAD

**Issue by btgh** [mantorok1/homebridge-omnilink-platform#31]
**Describe The Bug:**

The plugin "homebridge-omnilink-platform" requires Node.js version of ^14.18.2 || ^16.13.1 || ^18.12.1 which does not satisfy the current Node.js version of v20.9.0. 
**To Reproduce:**

upgrade to node v20
**Expected behavior:**

no warning message when using node v20
**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**: homebridge-omnilink-platform v1.5.11
* **Homebridge Version**: 1.7
* **Node.js Version**: 20.09
* **NPM Version**: 8.5.1
* **Operating System**: 
Ubuntu

**Comment by mantorok1** [comment]
Hi @btgh, I've released version 1.5.12 of the plugin with adds support for node 20.9

**Comment by btgh** [comment]
thank you - works great!
